data.sql

```diff
@@ -4,3 +4,11 @@ INSERT INTO animals VALUES(1, 'Agumon', '2020-02-03', 0, true, 10.23);
 INSERT INTO animals VALUES(2, 'Gabumon', '2018-11-15', 2, true, 8);
 INSERT INTO animals VALUES(3, 'Pikachu', '2021-01-07', 1, false, 15.04);
 INSERT INTO animals VALUES(4, 'Devimon', '2017-05-12', 5, true, 11);
+
+INSERT INTO animals VALUES(5, 'Charmander', '2020-02-08', 0, false, -11);
```

Comment:
[OPTIONAL]
- I noticed that you're hardcoding values into the `id` column. What do you think about not having to bother about what number is next to be inserted into the `id` column?(imagine you had ten million rows) 😉
You can add the `GENERATED ALWAYS AS IDENTITY` constraint to the `id` column. This ensures that sequential integers that cannot be altered are generated each time a record is being created. I hope it makes sense. 😄 
You can go through this [article](https://www.sqltutorial.org/sql-identity/#:~:text=The%20GENERATED%20ALWAYS%20generates%20sequential,integers%20for%20the%20identity%20column.).

Reply:
Enormous thanks for this tip.
I'm loving SQL 😄